src/main/java/ch/bfh/bti7081/s2018/green/NavigatorUI.java

```diff
@@ -173,7 +139,7 @@ private void initializeClasses() {
 
         DataContainer data = DataContainer.getInstance();
 
-        EntityManagerFactory emf = Persistence.createEntityManagerFactory("pmsDB");
+        /*EntityManagerFactory emf = Persistence.createEntityManagerFactory("pmsDB");
```

Comment:
Why comment this out? We need to set a current patient and staff or else the application doesn't work

Reply:
This has been commented out initially while troubleshooting (you remember I failed because of using JDK10 and then Hibernate dind't work). So I've added it back - just forgot to do this...